Src/BootCamp.Chapter/Demo.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    internal class Demo1
+    {
+        internal static void TwoDGrid()
+        {
+            Console.WriteLine("How much rows do you want the grid to be");
+            var input = Console.ReadLine();
+            var rows = ValidateInput(input);
+
+            var grid = new bool[rows, rows];
+            for (int i = 0; i < rows; i++)
+            {
+                for (int j = 0; j < rows; j++)
+                {
+                    grid[i, j] = false;
+                }
+
+            }
+
+            var cleaner = new GridCleaner();
+            var twoDGrid = new ToggleableGrid2D(grid, cleaner);
+            int x, y;
+            InputToggle(out input, out x, out y);
+
+            twoDGrid.Toggle(x, y);
+
+        }
+
+        private static void InputToggle(out string input, out int x, out int y)
+        {
+            Console.WriteLine("Give me the x-coordinate of a cell which you want to toggle");
+            input = Console.ReadLine();
+            x = ValidateInput(input);
+            Console.WriteLine("Give me the y-coordinate of a cell which you want to toggle");
+            input = Console.ReadLine();
+            y = ValidateInput(input);
+        }
+
+        internal static void JaggedArray()
```

Comment:
This is my last comment until you start paying attention . Please try to make methods verbs and type names nouns. You name however you like and don't seem to pay any attention to my remarks.

Reply:
sorry that you feel this way. I try to pay attention. Maybe I have to slow down and not make up a name too fast. Is maybe CreateJaggedArray a better name.